docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    =
+SPHINXOPTS    = -W --keep-going -n --color -w /tmp/sphinx-log
```

Comment:
I think the logfile shouldn't be here. That's only needed on the CI system

Reply:
done